plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/GitOutputPartViewImpl.java

```diff
@@ -136,4 +150,14 @@ public void scrollBottom() {
         scrollPanel.getElement().setScrollTop(scrollPanel.getElement().getScrollHeight());
     }
 
+    @NotNull
+    private void createButton(@NotNull SVGResource icon,
```

Comment:
Align it. Please use [formatter](https://github.com/codenvy/codenvy-dev-resources/blob/master/che-codestyle/src/main/resources/che-codestyle-intellij.xml) for java source files.